openshift_performance/ci/scripts/large_object_list/large_object_list.sh

```diff
@@ -0,0 +1,177 @@
+###############################################
+## Auth=lhorsley@redhat.com
+## Description: Cluster resilience testing via creation of a large number of secrets with a specific labelSelector in namespaces on an OWN cluster
+## Polarion test case: OCP-41643 - Large object list testing (Listing secrets in all namespaces with a specific labelSelector)	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-65206
+## Bugs related: https://bugzilla.redhat.com/show_bug.cgi?id=2050230 and https://bugzilla.redhat.com/show_bug.cgi?id=2094012
+## Cluster config: 3 master (m5.4xlarge or equivalent) with 40 workers (vm_type_workers: m5.2xlarge or equivalent).
+## The machine running the test should have at least 4 cores.
+## Note: While the test runs, check the functionality of the cluster. A simple example:
+##       while true; do oc get co --no-headers| grep -v 'True.*False.*False'; oc get nodes --no-headers| grep -v ' Ready'; date; sleep 10; done
+################################################ 
+
+#! /bin/bash
+
+expected_namespaces=${1:-100}
+secrets_per_namespace=${2:-10}
+expected_secrets=$(( $expected_namespaces * $secrets_per_namespace ))
+debug_secrets_command="oc get secrets -A -l test=listobject --no-headers -v9"
+my_secret_name="-secret-"
+my_namespace="objecttest-"
+my_object_type="namespaces"
+
+# simple function to display the status of operators and nodes
+function get_operator_and_node_status() {
+  echo ""
+  echo "Node and operator status check"
+  oc get nodes
+  echo ""
+  oc get co
+  echo ""
+}
+
+# function to create namespaces and secrets
+function create_namespaces_and_secrets() {
+
+  for ((i=0; i<$expected_namespaces; i++)); do
+      my_namespace_command='oc new-project --skip-config-write "${my_namespace}${i}" > /dev/null && echo -e "\n\nCreated project ${my_namespace}${i}" &>> object-list-test.log &'
```

Comment:
Lets try to use kube-burner or the cluster-loader script to create the objects here 

Think you can use kube-burner and use some of the common functions if you moved this to the [perfscale_regression_ci folder](https://github.com/openshift/svt/tree/master/perfscale_regression_ci/scripts/performance)

Could set _replicas_ = _secrets_per_namespace_ in the config file and _JOB_ITERATION_ = _expected_namespaces_ 

Example config that'll be very similar to what you need: https://github.com/openshift/svt/blob/master/perfscale_regression_ci/kubeburner-object-templates/blank-projects-config.yml

Reply:
In the Jira task, Qiujie stated the script should be done in a way a customer can run the test. @qiliRedHat What's your opinion?

Reply:
Customers should be able to run kube-burner since it is open source. 

How long does the creation of the objects take? With kube-burner it should be quicker and will help verify the objects created

I'm not super tied to how to run this, just want to stay up to date with the tooling we are using other places